internal/kubernetes/kubernetes.go

```diff
@@ -24,8 +24,10 @@ import (
 )
 
 const (
-	NodePoolLabelName = "humio.com/node-pool"
-	FeatureLabelName  = "humio.com/feature"
+	NodePoolLabelName        = "humio.com/node-pool"
+	FeatureLabelName         = "humio.com/feature"
+	PodMarkedForDataEviction = "humio.com/marked-for-data-eviction"
+	TopologyKubernetesZone   = "topology.kubernetes.io/zone"
```

Comment:
You don't need to create a constant for `TopologyKubernetesZone` but can just use the upstream const like we refer to in the `images/helper/main.go` as `corev1.LabelZoneFailureDomainStable`. It does look like `corev1.LabelZoneFailureDomainStable` was deprecated and replaced by `corev1.LabelTopologyZone` so that's probably the one to use.